README.md

```diff
@@ -316,6 +316,31 @@ setVerbose(True)
 setVerbose(False)
 ```
 
+
+### Output Files
+Batch will automatically handle your output files when the user assigns a file pattern and container url. 
+
+```R
+# Pushing output files
+storageAccount <- "storageAccountName"
+outputFolder <- "outputs"
+
+createContainer(outputFolder)
+writeToken <- rAzureBatch::createSasToken("w", "c", outputFolder)
+containerUrl <- rAzureBatch::createBlobUrl(storageAccount = storageAccount,
+                                           containerName = outputFolder,
+                                           sasToken = writeToken)
+
+output <- createOutputFile("test-*.txt", containerUrl)

```

Comment:
What kind of examples are you thinking of..? Also should we create a sample instead of a doc then?